src/mds/MDCache.cc

```diff
@@ -9484,6 +9484,19 @@ void MDCache::handle_find_ino(const cref_t<MMDSFindIno> &m)
   if (in) {
     in->make_path(r->path);
     dout(10) << " have " << r->path << " " << *in << dendl;
+
+    /*
+     * If the the CInode was just created by using openc in current
+     * auth MDS, but the client just sends a getattr request to another
+     * replica MDS. Then here it will make a path of '#INODE-NUMBER'
+     * only because the CInode hasn't be linked yet, and the replica
+     * MDS will keep retrying until the auth MDS flushs the mdlog and
```

Comment:
nit:`s/hasn't be/hasn't been` and `s/flushs/flushes`

Reply:
Done, thanks @neesingh-rh